### ISSUE TITLE
deletes files when moving save backup folder

### ISSUE THREAD

**Issue by imaxium78** [Razzmatazzz/RemnantSaveGuardian#97]
Moving backup save folder in settings when the folder is empty or before a backup has been made, causes RemnantSaveGuardian to delete folders and files in new backup save location. Also moves folders into new save folder location. reproduced once. 

I think its something to do with move files popup/command

in trying to reproduce, moving save backup location back and forth between two locations, and triggering move files popup, (using darksouls3 save files to test) it moves darksouls 3 save files folders but deletes ds3 files inside another folder. 
ie: save loc 1: games/back up save/remnant2 
https://imgur.com/ya6ni7r 
(test folder had 011 folder and graphics.xml in it)
save loc 2: user/documents/savebackups 

moving backupsave loc and saying yes move files, moves 011, remnant2, test folders but causes 011 folder inside test folder to be deleted.

initial issue deleted both 011 folders and put test folder inside remnant 2 folder. only reproduced that once.

TLDR: rip my original backup saves folders

**Comment by Razzmatazzz** [comment]
I'm not sure exactly what caused this, but I've modified that function to hopefully be a little safer. Fix will be in the next release.
